packages/backend-core/src/security/tests/permissions.spec.ts

```diff
@@ -17,10 +17,6 @@ describe("levelToNumber", () => {
     expect(permissions.levelToNumber(permissions.PermissionLevel.WRITE)).toBe(2)
   })
 
-  it("should return 3 for ADMIN", () => {
-    expect(permissions.levelToNumber(permissions.PermissionLevel.ADMIN)).toBe(3)
-  })
```

Comment:
How come this and the test below are being removed?

Reply:
This was causing a lot of confusion while trying to use the new "base" permission, and it should not be used. It used to be used before the app creator changes, that was a while ago.
But because cleaning it is not needed anymore, and at the risk of breaking some really old installations, I just undid these removals